### PR TITLE
MORE DOCS

### PR DESCRIPTION
**Pull request by DanielElisenberg**
### MORE DOCS
Documentation describing the metadatamodel, and validation done in this codebase.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+* **uriDefinition**: Lenke til ekstern ressurs med definisjon for variablen.
+* **unitType**: Se definisjonen under.
+* **valueDomain**: se definisjonen under.
+
+### MEASURE VARIABLES
+Beskrivelse av verdi-kolonnen for målte verdier. Det er en liste i modellen, men det støttes for øyeblikket kun 1 verdi-kolonne i microdata plattformen.
+* **shortName**: Teknisk navn på verdikolonnen. Brukes som variabelnavn i ROSE- klienten. Eksempel: INNTEKT_AKSJEUTBYTTE.
+* **name**: Navn(Tittel) på verdikolonnen. Brukes som kolonnenavn i ROSE-klienten. Eksempel: "Aksjeutbytte".
+* **description**: Beskrivelse av kolonnen. Eksempel: "Skattepliktig og skattefritt utbytte i... "
+* **dataType**: Datatype for verdiene. En av ["STRING", "LONG", "DOUBLE", "DA TE"]
```
Comment (vaskalan) [inline docs/README.md:34]: "DATE" :-)

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+12345678911;100000;2018-01-01;2018-12-31;
+12345678911;150000;2020-01-01;2020-12-31;
+```
+
+Dette er et eksempel på et dataset som beskriver personer sin inntekt over tidsperioder. Kolonnene kan beskrives slik:
+* Identifikator: fødselsnummer
+* Verdi: Akkumulert lønn i tidsperioden
+* Start: start på tidsperioden
+* Stopp: slutt på tidsperioden
+* Tom kolonne
```
Comment (vaskalan) [inline docs/README.md:135]: Hvorfor?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+### GENERELLE VALIDERINGSREGLER FOR DATA
+* Det kan ikke oppstå tomme rader i datasettet
+* En rad kan ikke være lenger en 5 elementer
```
Comment (vaskalan) [inline docs/README.md:139]: Hva er det 5.te element?

**Review by vaskalan — APPROVED** [review]

File: docs/README.md (hunk 3 above)
Comment (DanielElisenberg) [reply docs/README.md:139]: Godt spørsmål 👍🏻 har lagt til forklaring i den siste committen

**Comment by DanielElisenberg** [comment]
## UPDATES:
* more specific file deletes
* file structure now follows the structure of dataset-builder: inputdir/DATASET_NAME/DATASET_NAME.[csv|json]
* workingdir cleanup now targets only the files generated, decreasing the chance of unwanted deletes
* README updates for changes, and links to deeper docs in toplevel readme

**Pull request merged**